stdpopsim/cli.py

```diff
@@ -239,6 +239,7 @@ def get_citations(engine, model, contig, species):
     Return a list of all the citations.
     """
     citations = engine.citations[:]
+    citations.extend(species.genome.assembly_citations)
```

Comment:
@grahamgower isn't this all I need? seems to print fine

Reply:
Oh, yep. Sorry. Not sure how I missed that!